tests/integration/test_integration.py

```diff
@@ -155,3 +155,77 @@ def test_description():
             },
         ],
     )
+
+
+def test_description_narrative():
+    assert_table_contents(
+        "description_narrative",
+        expected_rows=[
+            {
+                "dataset": "test_activity",
+                "_link": "1.description.0.narrative.0",
+                "_link_activity": "1",
+                "_link_description": "1.description.0",
+                "prefix": "test_prefix",
+                "iatiidentifier": "AA-AAA-123456789-ABC123",
+                "reportingorg_ref": "AA-AAA-123456789",
+                "_": "General activity description text. Long description of the activity with no\n        particular structure.",  # noqa: E501
```

Comment:
A more descriptive column name would probably be better for users, maybe something like `narrative_text` or `narrative_content`?